Airports/EGDY/SMR/GEO.txt

```diff
@@ -0,0 +1,21 @@
+; Start Yeovilton EGDY GEO
+EGDY Yeovilton               S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
+; - Hold_N_08
```

Comment:
For single line geo, the comment should be placed at the end of the line:

e.g. `N051.00.28.470 W002.39.42.559 N051.00.28.420 W002.39.41.743 smrYellow ;Hold_N_08`

Reply:
Changed! Thanks